### PR TITLE
Content not updateable

### PR DESCRIPTION
**Pull request by lorcan-codes**
[Task details](https://www.notion.so/yuki-labs/Content-not-updateable-in-cms-972c31f8d9d540bfa556127204519fd8)
[Preview link](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en)

Review changes
1. Home page hero text
[Homepage SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/home)

2. Footer text
- [Footer text SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/footer)

3. Language center
- [Language center CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/language)

4. Search
- [Search SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/search)

5. Events title, subtitle 
[Events page](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en/events)
[Events past page](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en/events/past)
[Events SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/events)

6. Tutorials page
- [Tutorials page](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en/tutorials)
- [Tutorials course page](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en/tutorials/bytesized_series)
- [Tutorials SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/tutorials)

7. Jobs page
- [Jobs page](https://starknet-website-git-content-not-updateable-yuki-labs.vercel.app/en/jobs)
- [Jobs SEO CMS](https://2153223c.starknet-netlify-cms.pages.dev/#/collections/seo/entries/jobs)

### CODE REVIEW

File: workspaces/cms-scripts/src/index.ts [hunk 1]
```diff
@@ -45,32 +45,43 @@ for (const simpleData of simpleDataTypes) {
...
       const data = simpleFile.localeMap.get(locale)!;
 
       await write(
-        `_crowdin/data/${simpleFile.resourceName}/${locale}.json`,
-        data.items,
+        `_crowdin/data/${resourceDir}/${locale}.json`,
+        simpleFile.writeRootData ? data : data.items
       );
     }
   } catch (err) {
```
Comment (thecotne) [inline workspaces/cms-scripts/src/index.ts]: let's remove try catch (script should just fail if there is an error)

**Review by thecotne — CHANGES_REQUESTED** [review]
left one comment 

looks good overall 👍

**Review by robwalsh21 — APPROVED** [review]
Nice work @lorcan-codes

File: workspaces/cms-scripts/src/index.ts (hunk 1 above)
Comment (lorcan-codes) [reply workspaces/cms-scripts/src/index.ts]: @thecotne Done!

**Review by thecotne — APPROVED** [review]

**Pull request merged**